tests/res_diff_expect/test_result_both_fail.ion

```diff
@@ -0,0 +1,20 @@
+$ion_1_0
+{
+ test: {
+  read_error: {
+   message: "Read_error: new commit causes new read error(s).",
```

Comment:
I'm thinking we should just show the complete change. Maybe change the message to `Read_error: new commit has different read errors(s).` and include the errors from both revisions here. That will make it easier for the user to evaluate the difference.

Reply:
Added. We now write all difference into the report but only alert user once the new commit has errors that the old didn't have.